README.md

```diff
@@ -1 +1,55 @@
-# portfolio-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Basic portofolio project
+
+> This is the very first project at Microverse that initiate students on how to use Git/Github and Linting for their projects
+
+![screenshot](./src/assert/images/mobile1.PNG)
+
+## Built With
+
+- HTML and CSS
+
+## Getting Started
+
+**In this project, students at Microverse will get familiar with CSS, HTML, git/github/github flow and linting**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+#### Have nodejs an npm installed locally
+
+### Setup
+
+#### To get the copy of this project, please go ahead and clone the project from the source repository and run `git clone https://github.com/mwafrika/portfolio-microverse.git`
+
+#### Next step is to open the cloned repository and run the following command to install the project dependencies locally `npm install` in
+
+### Usage
+
+#### run the following command to fix the linting errors
+
+` npm run fix`

```

Comment:
- [ ] As the previous reviews suggested please remove this part. Linters are a tool for you as a developer so you don't need to specify how to run them or how to fix linter problems in your README.
- [ ] I think when the first reviewer told you " Please, modify your README file with how to set up your project, not the linter." He meant that you should add some steps for anyone that wants to run your project locally. 
## Example: 
![image](https://user-images.githubusercontent.com/28109626/148425619-006fd476-e60b-43dd-807b-a5d215171f72.png)

 Please let's fix that  :+1: